examples/provider/provider.tf

```diff
@@ -28,6 +28,7 @@ resource "pritunl_user" "test" {
   name            = "test-user"
   organization_id = pritunl_organization.developers.id
   email           = "test@test.com"
+  pin             = "123456"
```

Comment:
Create an additional example instead of modifying this one. Pin usage is an optional (but secure) way for users management

Reply:
Done.